src/ethereum_test_vm/helpers.py

```diff
@@ -0,0 +1,20 @@
+"""
+Helper functions for the EVM.
+"""
+
+from .opcode import Opcodes as Op
+
+
+def call_return_code(opcode: Op, success: bool, *, revert: bool = False) -> int:
+    """
+    Returns the return code for a CALL operation.
+    """
+    if opcode in [Op.CALL, Op.CALLCODE, Op.DELEGATECALL, Op.STATICCALL]:
+        return int(success)
+    elif opcode in [Op.EXTCALL, Op.EXTDELEGATECALL, Op.EXTSTATICCALL]:
+        if success:
+            return 0
+        if revert:
+            return 2
```

Comment:
just scanning quickly, but noticed this: revert is code 1 not 2. 2 is for exceptional failure, i.e. oog or Op.INVALID

Reply:
Must have been too tired yesterday since I swear I read the opposite from the EIP :sweat_smile: